### DISCUSSION TITLE
Mobx data fetching query client ? :)

### DISCUSSION THREAD

**Discussion by WoodenPC** [mobxjs/mobx#3827] [Ideas]
Hi, is it possible to implement data fething library with full integration with mobx like https://github.com/posva/pinia-colada for pinia (vue.js) ?

**Comment by kubk** [comment]
You may be interested in the following examples:
- Mobx + React Query integration: https://www.youtube.com/watch?v=rNuIyo994OY
- Cache library for MST: https://github.com/ConrabOpto/mst-query
